kaybee/internal/model/statement.go

```diff
@@ -108,6 +105,16 @@ func (n Note) String() string {
 	return output + "\n"
 }
 
+// ContainsSlice tells whether a contains x.
+func ContainsSlice(a []string, x string) bool {
```

Comment:
Functionally identical to previous solution, but now the name does not reflect the fact that it only operates on string slices.... (how would you name a similar function that operates on slices of statements?)

I know generics are coming to the rescue, but in the meantime.... ;-)

Reply:
I agree, `StringSliceContains` would be a better function name.

Reply:
right